spec/views/users/index.html.erb_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe "users/index", type: :feature do
+  before(:each) do
+    User.create!(
+      name: "Jackie",
+      email: "Email",
+      password: 'a',
+      password_confirmation: 'a'
+    )
+    User.create!(
+      name: "Dane",
+      email: "Email2",
+      password: 'a',
+      password_confirmation: 'a'
+    )
+  end
+
+  it "renders a list of users" do
+    visit users_path
+    expect(page).to have_content('Jackie')
+    expect(page).to have_content('Dane')
+  end
+end
```

Comment:
This was a test I had to rewrite almost entirely to fit the way in which I create users.  Because there's a `password` and a `password_confirmation`, and just a `password_digest`, the users have to be created like this.